components/molecule/breadcrumb/src/utils.scss

```diff
@@ -0,0 +1,50 @@
+$breakpoints: (
```

Comment:
Mmmm I think breakpoint-up is still valid in the next sui-theme: https://github.com/SUI-Components/sui-theme/blob/cf07c47c68541bdb8a2f550235e96fe52bd34d83/src/utils/_breakpoints.scss#L18

Why exactly do you need this utils file?

Reply:
I'm getting this error if I remove the utils file

<img width="1437" alt="Captura de pantalla 2019-06-03 a las 14 47 23" src="https://user-images.githubusercontent.com/422576/58799669-9345cc80-860e-11e9-9817-47478723d801.png">

I've put in this file all functions needed for the current code so I can remove the dependency without errors

Reply:
you cannot use media-breakpoint-`down`, this task is about refactor in order to use media-breakpoint-`up` and make the component mobile first

Reply:
oks, fixed!